tests/polkadotjs_test/test/test-polkadot.js

```diff
@@ -0,0 +1,149 @@
+const {describe} = require('mocha');
+const {expect} = require('chai');
+const { ApiPromise, WsProvider } = require('@polkadot/api');
+const sleep = (delay) => new Promise((resolve) => setTimeout(resolve, delay))
+const {Keyring } = require('@polkadot/keyring');
```

Comment:
Please use standard `{ Prop }` spacing for these imports 

Additionally, please move Line 4 down, style guide wise all dependencies come first followed by declarations

Reply:
Updated, thanks for guidance regarding javascript style, I imagine there is a lot I may be missing here.  I'm going to research and start using a linting utility, do you have any suggestions?  Seems like a lot are using ESLint.